# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/magicpoint/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mng.c.orig	2004-05-14 11:27:46.980807688 +0200
+++ ./mng.c	2004-05-14 11:28:40.725637232 +0200
@@ -216,7 +216,7 @@
 	mymng = (mngstuff*)mng_get_userdata(mng);
 
 	/* pull out the chuck type as a string */
-	// FIXME: does this assume unsigned char?
+	/* FIXME: does this assume unsigned char? */
 	chunk[0] = (char)((chunktype >> 24) & 0xFF);
 	chunk[1] = (char)((chunktype >> 16) & 0xFF);
 	chunk[2] = (char)((chunktype >>  8) & 0xFF);
@@ -317,7 +317,7 @@
 	}
 }
 
-void 
+int
 mng_window_setup(mymng, x, y, width, height)
 	mngstuff	*mymng;
 	int x, y;
--- ./scanner.l.orig	2004-05-14 11:25:56.259639864 +0200
+++ ./scanner.l	2004-05-14 11:26:57.899269216 +0200
@@ -59,7 +59,7 @@
 #include "tokdefs.h"
 
 static int stoi __P((char *));
-static inline int xdtoi __P((int));
+static int xdtoi __P((int));
 
 #ifdef FLEX_SCANNER
 #define YY_NO_UNPUT
@@ -202,7 +202,7 @@
 }
 
 /* Hex digit to integer. */
-static inline int
+static int
 xdtoi(c)
 	register int c;
 {
